.cookiecutter/includes/requirements/dev.in

```diff
@@ -0,0 +1 @@
+h_testkit
```

Comment:
```suggestion
h-testkit
```

Should be `h-testkit` I think, but why is this needed in `dev.in`? The cookiecutter will automatically add this to `tests.in` and `functests.in`

Reply:
Ah, it looks like they are both equivalent:

https://packaging.python.org/en/latest/specifications/name-normalization/#name-normalization

Change it over the `-` version for consistency :+1: 

We added this to dev because `make shell` depends on it